recipes/pyspark/pysparkretailapp/data_saver.py

```diff
@@ -0,0 +1,47 @@
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2019 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+from sdk.data_saver import DataSaver
+
+class MyDatasetSaver(DataSaver):
+
+    def save(self, configProperties, prediction):
+        sparkContext = prediction._sc
```

Comment:
pylint reports, `Access to a protected member _sc of a client class (protected-access)`